ctapipe/reco/preprocessing.py

```diff
@@ -48,6 +56,17 @@ def check_valid_rows(table: Table, warn=True, log=LOG) -> np.ndarray:
     return valid
 
 
+def table_to_X(table: Table, features: list, log):
```

Comment:
type hint could be `List[str]`

Reply:
and could also be given for `log`

Reply:
Do you mean `log=LOG` (and `LOG=logging.getLogger(__name__)`) as for `check_valid_rows`, or do you mean `log: List[str]`?

Reply:
The type of log is `logging.Logger`, so it should be `log: logging.Logger`.

Reply:
I think passing a default would be better, to keep it consistent with the other functions.